src/wazuh_qa_framework/system/wazuh_handler.py

```diff
@@ -369,9 +400,36 @@ def configure_host(self, host, configuration_host):
                                     'value': 121.1.3.1
         Args:
             host (str): Hostname
-            configuration_host (Map): Map with new hosts configuration
+            configuration_file (str): File name to be configured
+            configuration_values (dict): Dictionary with the new configuration
         """
-        pass
+        self.logger.debug(f"Configuring {configuration_file} in {host}")
+
+        if configuration_file not in conf_functions:
+            raise Exception(f"Invalid operation for {configuration_file} configuration file. Please select one \
+                              of the following: {conf_functions.keys()}")
+
+        # Get group folder and new configuration for agent.conf
+        group = configuration_values.get('group', 'default') if configuration_file == 'agent.conf' else None
+        configuration_values = (configuration_values['configuration'] if configuration_file == 'agent.conf'
+                                else configuration_values)
+
+        # Get configuration file path
+        host_configuration_file_path = self.get_file_fullpath(host, configuration_file, group)
+
+        parameters = {'new_conf': configuration_values}
+
+        # Get template for ossec.conf and agent.conf
+        if configuration_file == 'ossec.conf' or configuration_file == 'agent.conf':
```

Comment:
```suggestion
        if configuration_file in ['ossec.conf', 'agent.conf']:
```
We can also move that list into a variable called xml_configuration_files

Reply:
Done in 1deaf31